contents/blog/the-posthog-array-1-35-0.md

```diff
@@ -0,0 +1,109 @@
+---
+date: 2022-04-25
+title: "Array 1.35.0: SAML, maps and plugins"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes", "Product updates"]
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: PostHog 1.35.0 introduces activity logs and a brand new way of visualizing where your users are coming from with the World Map. Additionally we now support organization-level SAML login on both Cloud and Self-Hosted instances. Plus check out your Project Homepage for a few additional goodies.
+---
+
+PostHog 1.35.0 introduces audit logs and a brand new world map view to help you visualize where your users are coming from. Additionally, we now support organization-level SAML login on both Cloud and Self-Hosted instances. Plus, check out your project homepage for a few additional goodies!
+
+We've also recently launched a new newsletter, which we send once every two weeks! [Subscribe today](/newsletter) to hear about our latest news, what we're reading and how to get the most out of PostHog. Each issue also includes a bonus hedgehog cartoon!
+
+<blockquote class='warning-note'>
+<b>IMPORTANT!</b> If you use SAML, please read our <a href="deprecation-and-removal-notices">deprecation notices below</a>.
+</blockquote>
+
+## PostHog 1.35.0 release notes
+
+> Wondering how to upgrade a self-hosted instance? Check out our [upgrade guide](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Activity log](#new-activity-log)
+- [World map](#new-world-map)
+- [Multi-tenant SAML](#new-multitenant-saml)
+- [Project homepage updates](#improvement-project-homepage-updates)
+- [New Avo plugin](#new-avo-plugin)
+
+### New: Activity log
+<img width="827" alt="A screenshot of the app showing the new 'history' tab on the feature flags page. In the screenshot you can see three recent changes by PostHog staff to our feature flags" src="https://user-images.githubusercontent.com/984817/164484091-f5dfce0b-c400-4699-ac55-642ccc2bc55b.png" />
+
+Ever wondered who deleted that feature flag, or merged those two people? Wonder no more with the new Activity Log™️!
```

Comment:
```suggestion
Ever wondered who deleted that feature flag, or merged those two people? Wonder no more with the new activity log!
```

Charles has asked us to remove the TM.

Reply:
@charlescook-ph can we say "patent pending" :)

Reply:
haha, oh Paul you joker

but no, we can't 🧑‍⚖️ 🚓

Reply:
😢